checkmate/views/status.py

```diff
@@ -1,10 +1,10 @@
-"""The status end-point required to allow us to work with AWS."""
-
```

Comment:
This is why I don't think we should force ourselves to add docstrings to everything even when they're not really necessary: this docstring is misleading. First of all, it's not just used by "AWS" but by Elastic Beanstalk specifically. Secondly, that's not its only use, for example it's also used by Pingdom and could be used by other things as well.

Reply:
Yup, we might miss some docstrings without enforcing it but that's a job for the PR reviews IMO.

Enforcing it leads to a lot of placeholder-like messages.